path_login.go

```diff
@@ -178,15 +178,23 @@ func (b *azureAuthBackend) verifyClaims(claims *additionalClaims, role *azureRol
 		return fmt.Errorf("token is not yet valid (Token Not Before: %v)", notBefore)
 	}
 
+	if (len(role.BoundServicePrincipalIDs) == 1 && role.BoundServicePrincipalIDs[0] == "*") &&
+		(len(role.BoundGroupIDs) == 1 && role.BoundGroupIDs[0] == "*") {
+		return fmt.Errorf("expected a specific BoundGroupID or BoundServicePrincipalID; both cannot be '*'")
```

Comment:
I suggest to use the parameter names that are familiar to users instead of the Go variable names in the error message. 

Something like:
```
"expected specific bound_group_ids or bound_service_principal_ids; both cannot be '*'"
```

Reply:
Ah, good call. Added